src/styles/blocks/button.scss

```diff
@@ -0,0 +1,20 @@
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  &:hover {
+    background: #fff;
+    border: 1px solid #00acdc;
+    color: #00acdc;
```

Comment:
Changing the 'background' and 'color' properties on hover might cause the text to become unreadable if the button is placed on a background with a similar color to '#00acdc'. Consider maintaining sufficient contrast for accessibility.